drivers/unifiedPlatform/pds/backend/apiv1/restore.go

```diff
@@ -0,0 +1,113 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"github.com/jinzhu/copier"
+	pdsv2 "github.com/portworx/pds-api-go-client/unifiedcp/v1alpha1"
+	"github.com/portworx/torpedo/drivers/unifiedPlatform/apiStructs"
+	"github.com/portworx/torpedo/drivers/unifiedPlatform/utils"
+	status "net/http"
+)
+
+// getRestoreClient updates the header with bearer token and returns the new client
+func (ds *PDSV2_API) getRestoreClient() (context.Context, *pdsv2.RestoreServiceAPIService, error) {
+	ctx, token, err := utils.GetBearerToken()
+	if err != nil {
+		return nil, nil, fmt.Errorf("Error in getting bearer token: %v\n", err)
+	}
+	ds.ApiClientV2.GetConfig().DefaultHeader["Authorization"] = "Bearer " + token
+	ds.ApiClientV2.GetConfig().DefaultHeader["px-account-id"] = ds.AccountID
+	client := ds.ApiClientV2.RestoreServiceAPI
+
+	return ctx, client, nil
+}
+
+// CreateRestore will create restore from a given backup
+func (ds *PDSV2_API) CreateRestore(createRetsoreRequest *apiStructs.WorkFlowRequest) (*apiStructs.WorkFlowResponse, error) {
+	restoreResponse := apiStructs.WorkFlowResponse{}
+
+	_, restoreClient, err := ds.getRestoreClient()
+	if err != nil {
+		return nil, fmt.Errorf("Error in getting context for backend call: %v\n", err)
+	}
+
+	restoreModel, res, err := restoreClient.RestoreServiceCreateRestoreExecute(createRetsoreRequest.Restore.Create.V1)
+	if err != nil && res.StatusCode != status.StatusOK {
+		return nil, fmt.Errorf("Error when calling `RestoreServiceCreateRestoreExecute`: %v\n.Full HTTP response: %v", err, res)
+	}
+	copier.Copy(&restoreResponse, restoreModel)
```

Comment:
Can we handle the copier error as well. It will help in debugging in case of failure. Applies to all instances.